hrm-service/src/contact-job/contact-job-complete.ts

```diff
@@ -71,6 +74,8 @@ export const pollAndprocessCompletedContactJobs = async (jobMaxAttempts: number)
               'Attempts limit reached',
             );
 
+            await deleteCompletedContactJobsFromQueue(m.ReceiptHandle);
```

Comment:
Do we want to remove this from the queue if the previous database steps fail? I assume it will just keep retrying if not.

It's ok as is, but something to think about - if we can have this as a `finally` action if we want HRM to handle the timeout / retry rather than retrying it from the queue

Reply:
After giving some more thought to this, I am not sure we want to delete the message if something's going on with the db. If so, doesn't the same apply for the success case? :thinking: 
Merging the PR as is now, but will open a follow up to address this (I'll reach you first :P).